src/routes/docs/tutorials/nuxt/step-8/+page.markdoc

```diff
@@ -0,0 +1,11 @@
+---
+layout: tutorial
+title: Next steps
+description: View your Nuxt app built on Appwrite Cloud.
+step: 8
+---
+
+# Test your project {% #test-project %}
+Run your project with `npm run dev -- --open --port 3000` and open [http://localhost:3000](http://localhost:3000) in your browser.
```

Comment:
In another one of the steps, we mentioned using `npm run dev` directly, instead of specifying a port.

IMO, we should not try and change the port, no reason to. Port 3000 is a React convention, most projects use something else, and it's fine :)